R/init.R

```diff
@@ -228,7 +238,9 @@ init <- function(data,
       if (!is.null(landing_module)) {
         do.call(landing_module$server, c(list(id = "landing_module_shiny_id"), landing_module$server_args))
       }
-      srv_teal_with_splash(id = id, data = data, modules = modules, filter = deep_copy_filter(filter))
+      srv_teal_with_splash(
+        id = id, data = data, modules = modules, filter = deep_copy_filter(filter), lockfile_task = lockfile_task
+      )
```

Comment:
would it be possible to move this inside `srv_teal_with_splash` so that we don't change its arguments? This is something that I would like to avoid if possible

Reply:
actually even into the `srv_teal()`

Reply:
@pawelru yeah, I think it is possible!

Reply:
I just wanted to start the process as soon ass possible - that's why it's in init.

Reply:
90fbadbced4bc79959caba85684a33c0fb9382cd